ansible/playbooks/roles/prometheus/tasks/upgrade/main.yml

```
@@ -0,0 +1,122 @@
+---
+# General things needed 
+- name: Prometheus | Include new version default variables
```

```suggestion
- name: Include new version default variables
```
We use `{{ component }} |` prefix for tasks in `upgrade` role since Ansible prints just current role name so there is no information about current component. Here tasks are inside `prometheus` role so prefix is unneeded.

---

Gotcha, should be fixed.